config/config.go

```diff
@@ -211,8 +212,13 @@ func validateReferencesExist(c *configpb.Configuration) error {
 // TestGrid file prefix, post-normalization.
 func validateName(s string) error {
 	name := normalize(s)
-	if name == "" {
-		return errors.New("normalized name can't be empty")
+
+	if len(name) < 3 {
+		return errors.New("names must contain at least 3 alphanumeric characters")
+	}
+
+	if len(name) > 2048 {
```

Comment:
Add unit test?

I would make these consts for a more robust unit test, aka:

```golang
const small = 3, big = 4

cases := []struct{
  name string
  n int
  pass bool
}{
  {
    name: "too small fails",
    n: small-1,
    err: true,
  },
  {
    name: "small pass",
    n: small,
    err: true,
   },
   // same for big and big+1
}
```

also

```golang
var sb strings.Builder
for i := 0; i<tc.n;i++ { 
  sb.WriteRune('a')
}
name := sb.String()
```

Reply:
(AKA that way we can change big from 2048 to 1024 or whatever without updating the unit test)
(Also fine to just use the 2048 literal in the unit test)

Reply:
This feels like overkill, but it does give me an opportunity to make existing tests a bit more descriptive.